discovery_engine_core/core/src/storage.rs

```diff
@@ -100,18 +122,19 @@ pub mod models {
         }
     }
 
+    #[derive(Debug)]
     #[allow(dead_code)]
     pub(crate) struct ApiDocumentView {
         pub(crate) document_id: document::Id,
         pub(crate) news_resource: NewsResource,
         pub(crate) newscatcher_data: NewscatcherData,
         pub(crate) user_reacted: Option<UserReaction>,
-        // //FIXME I don't think this is helpful as multiple documents in the vec can have the same value for this!
+        // FIXME I don't think this is helpful as multiple documents in the vec can have the same value for this!
```

Comment:
:pushpin: This is outdated I had at some point realized how this is used and what it's for which is how the `PresentationOrdering` was created.

The reason I'm putting a :pushpin: here is that at some point we should probably move the timestamp out of `NewsResource` and combine it with the `in_batch_index` to create a `PresentationOrdering` type.

But I think this is probably best done after we fully moved to using the sql database.